src/lib/Event/ContentProxyCreateEvent.php

```diff
@@ -15,7 +15,7 @@
 /**
  * @internal
  */
-final class ContentProxyCreateEvent extends Event
+class ContentProxyCreateEvent extends Event
```

Comment:
This rather should have a "mockable" contract, but since it's not injectable, I won't insist ;)